tests/integration/misc-tosca-types/runme.sh

```diff
@@ -6,9 +6,13 @@ opera_executable="$1"
 
 # perform integration test with service template
 $opera_executable validate -i inputs.yaml service-template.yaml
+$opera_executable info
```

Comment:
I'd really like it if you added at least one test for the contents, eg

```suggestion
info_out="$($opera_executable info --format json)"
test "$(echo "$info_out" | jq -r .status)" = "null"
# and other tests
```

Reply:
Ok, I added a few of these lines to the script.